CONTRIBUTING.md

```diff
@@ -3,6 +3,10 @@
 The SPDX specification is maintained by the SPDX [legal][spdx-legal] and [tech][spdx-tech] teams.
 Design and planning is primarily done via the team [mailing][spdx-legal-list] [lists][spdx-tech-list] and meetings.
 
+## Contribution License Agreement
+
+Contributions to this repo are made pursuant to the [SPDX Community Specification Contributor License Agreement 1.0][cla]. You do not need to submit a signed copy of the contributor license agreement; by making a contribution to this repo, you agree to the terms set forth in that agreement.
```

Comment:
Propose to change" by making a contribution to this repo," -> "by making a contribution to this repo with signed-off commits",

Reply:
My understanding of the way the Community Specifications model works is that it is intended to cover any contributions to the repo -- not only commits, but also discussions on conversation threads, etc.

As a reference point, this is comparable to the text of the [Apache-2.0 license](https://www.apache.org/licenses/LICENSE-2.0), emphasis added:

> "Contribution" shall mean any work of authorship, including the original version of the Work and any modifications or additions to that Work or Derivative Works thereof, that is intentionally submitted to Licensor for inclusion in the Work by the copyright owner or by an individual or Legal Entity authorized to submit on behalf of the copyright owner. For the purposes of this definition, **"submitted" means any form of electronic, verbal, or written communication sent to the Licensor or its representatives, including but not limited to communication on electronic mailing lists, source code control systems, and issue tracking systems** that are managed by, or on behalf of, the Licensor for the purpose of discussing and improving the Work . . .

Reply:
(so to clarify, no, I don't think this should be changed.)

Reply:
See my comment below Community-Spec-1.0 does not require DCO but our [SPDX Community Specification CLA](https://github.com/spdx/governance/blob/main/0._SPDX_Contributor_License_Agreement.md)  does.